spec/20-http_header_format.md

```diff
@@ -394,9 +391,9 @@ Platforms and tracing systems MUST propagate this field. There might be multiple
 system may propagate them as they came, combine into a single header or split
 into multiple headers differently following the RFC specification.
 
-Platforms and tracing vendors SHOULD propagate at least 512 characters of a
-combined header. This length includes commas required to separate list items.
-But does not include optional white space (`OWA`) characters.
+Platforms and tracing vendors _SHOULD_ propagate at least 512 characters of a
```

Comment:
we don't use italic for MUST/SHOULD words, capitalization is enough.

Reply:
We decided that we will not be using italics going foward, see [this comment](https://github.com/w3c/trace-context/pull/307#issuecomment-511920186), but will address removal in a future PR.